tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -41,6 +49,7 @@ def tearDown(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, pdu_co
                           'Recover {} by PDU reboot failed'.format(hostname))
             # Wait until all critical processes are healthy.
             wait_critical_processes(duthost)
+            wait_lc_healthy_if_sup(duthost, duthosts)
```

Comment:
is wait_lc_healthy_if_sup in class TestMemoryExhaustion?

Reply:
thanks for reminding! done